routes/views/login.js

```diff
@@ -11,30 +12,42 @@ exports = module.exports = function (req, res) {
   locals.section = 'login';
 
   view.on('post', function (next) {
-    console.log('[routes/login.js] Login post function executed')
     const email = req.body.inputEmail;
     const password = req.body.inputPassword;
 
     // firebase auth method referred from - 
     // https://firebase.google.com/docs/reference/js/firebase.auth.Auth#signInWithEmailAndPassword
     firebase.auth().signInWithEmailAndPassword(email, password)
       .then(function (user) {
-        const uid = user.user.uid
-        console.log("User UID : " + uid)
+        const uuid = user.user.uid
+        console.log("User UID : " + uuid)
+        console.log('[routes/login.js] Login post function executed')
+
+        /*
+        difference between findOne() and find():
+        findOne - if query matches, first document is returned, otherwise null.
+        find - nomatter number of documents matched, a cursor is returned, never null.
+        */
+        customer.model.findOne({ uid: uuid }).exec(function (err, customer) {
+          if (err) {
+            console.log(err);
+            res.status(500).send('DB Error');
+            next();
+          }
```

Comment:
below condition - if (!customer) - will never be executed because "no user exist" error is already caught by firebase signIn function above.